### ISSUE TITLE
az cli failing to run commands

### ISSUE THREAD

**Issue by ander1044** [Azure/azure-cli#21305]
## Describe the bug

**Command Name**
`az postgres server firewall-rule create`

**Errors:**
```
No module named 'azure.mgmt.rdbms.postgresql.operations'
Traceback (most recent call last):
python3/dist-packages/knack/cli.py, ln 233, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 558, in execute
    self.commands_loader.load_arguments(command)
...
<frozen importlib._bootstrap>, ln 1030, in _gcd_import
<frozen importlib._bootstrap>, ln 1007, in _find_and_load
<frozen importlib._bootstrap>, ln 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'azure.mgmt.rdbms.postgresql.operations'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az postgres server firewall-rule create --name {} --server-name {} --resource-group {} --start-ip-address {} --end-ip-address {} --subscription {}`

## Environment Summary
```
Linux-5.14.0-9parrot1-amd64-x86_64-with-glibc2.31
Python 3.9.2
Installer: DEB

azure-cli 2.18.0 *

Extensions:
azure-devops 0.19.0

```

**Comment by yonzhan** [comment]
route to CXP team

**Comment by ander1044** [comment]
SOLVED: installed Azure-cli version 2.29.0.
